### PR TITLE
Update WorldEnvironment.xml

### PR DESCRIPTION
**Pull request by abhaskumarsinha**
Added Some more description about that node.

Correct the Grammar Mistake (if any)

### CODE REVIEW

File: doc/classes/WorldEnvironment.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+		The [code]WorldEnvironment[/code] node can be added to a scene in order to set default [Environment] variables for the scene. The [code]WorldEnvironment[/code] can be overridden by an [Environment] node set on the current [Camera]. Additionally, only one [code]WorldEnvironment[/code] may be instanced in a given scene at a time. The [code]WorldEnvironment[/code] allows the user to specify default lighting parameters (e.g. ambient lighting), various post-processing effects (e.g. SSAO, DOF, Tonemapping), and how to draw the background (e.g. solid color, skybox). Usually, these are added in order to improve the Realism/Color Balance of the Scene. Godot 3.0 supports Paranomas to be set as the background of a Scene. Without using Paranoma Image, the Sky color and Sun can be Placed in the Scene.
```
Comment (guilhermefelipecgs) [inline doc/classes/WorldEnvironment.xml]: Change ```Paranoma``` to ```Panorama```.

Comment (guilhermefelipecgs) [reply doc/classes/WorldEnvironment.xml]: And I don't think it's necessary write some letters with uppercase in the same sentence like this:
```the Sky color and Sun can be Placed in the Scene```

This way is ok:
```the sky color and sun can be placed in the scene```

Comment (StraToN) [reply doc/classes/WorldEnvironment.xml]: Same remarks as @guilhermefelipecgs: 
useless capital letters in `Realism/Color Balance` for example.

**Comment by akien-mga** [comment]
Merged manually as 1a1303a94 with some style edits and simplifications. (I removed the part about panoramas as it's best documented directly in [Environment] and [Sky]).

**Pull request closed without merging**